src/ndb_p3.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2016 Swift Navigation Inc.
+ * Contact: Roman Gezikov <rgezikov@exafore.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#include <string.h>
+#include <libswiftnav/logging.h>
+#include "ndb.h"
+
+#define GPS_L2C_CAPB_FILE_NAME "l2capb"
+static u32 gps_l2c_capabilities _CCM;
+static ndb_element_metadata_t gps_l2c_capabilities_md _CCM;
+
+static ndb_file_t gps_l2c_capb_file = {
+  .name = GPS_L2C_CAPB_FILE_NAME,
+  .fh = -1,
+  .expected_size = sizeof(ndb_file_version) +
```

Comment:
If every file will include version information, I would abstract that away in ndb.c. No reason the individual sections need to know about it.